cadcdata/cadcdata/storageinv.py

```diff
@@ -487,16 +485,11 @@ def cadcput(self, id, src, replace=False, file_type=None,
         :param id: unique identifier (URI) for the file in the CADC inventory
         system. The URI must include the scheme.
         :param src: location of the source file
-        :param replace: boolean indicated whether this is expected to be
-        a replacement of an existing file in the inventory system or not (file
-        is new). Wrong assumption results in an error. This is a safeguard
-        for accidental file replacements.
+        :param replace: remains for backwards-compatibility, value ignored.
```

Comment:
```suggestion
        :param replace: DEPRECATED - value ignored. It will be removed in future versions.
```
Not sure whether we should issue a warning when the attribute is used. For that the default should be set to `None` and issue a warning in case it comes with a value.